config/init.go

```diff
@@ -76,7 +76,7 @@ func InitWithIdentity(identity Identity) (*Config, error) {
 			APICommands: []string{},
 		},
 		Reprovider: Reprovider{
-			Interval: "12h",
+			Interval: "22h",
 			Strategy: "all",
```

Comment:
If we want this change to be effective, this needs both a refactor and a config migration.

1. Both fields defined in 
    https://github.com/ipfs/kubo/blob/83b3ec4a61f086f81e0f75c5ec9d508bf5385b16/config/reprovider.go#L4-L5
    should be replaced with `optionalString` variants, and the implicit default should no longer be hardcoded in user config
2. Config migration should be added, and it should migrate users running with default values to implicit defaults
    - If Interval = 12h and Strategy = all, then remove both from config (switch to implicit default added in step 1)

Reply:
Good callout @lidel .

Is this approachable for a non-maintainer to do?  @cortze can you take this?

Reply:
I could take a look at it if that is okey for you, will come back if there are any questions :)